src/scenes/DeliverablesRouter/DeliverableView.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
Should we merge this with the 'DeliverablesView' file in the same directory?
Or rename the other file to something else?

Reply:
I will rename it, I think something like `DeliverableViewWrapper` makes sense